apiary.apib

```diff
@@ -2881,6 +2881,66 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
+
+*Required scopes: `projects`*
+
+### projectItems.list [GET /projectItems.list]
+
+Get a list of project items.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, required)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + date
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+                        + desc
+            + Default
+                + (object)
+                    + field: `date`
+                    + order: `desc`
+
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+                + project (object)
+                    + type: `project` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + type: `time_tracked`
+                + description: `Software license` (string)
```

Comment:
If you use "time_tracked" as the example, I would not use "Software license" as the description. Rather some realistic description for time tracking, like "Drafted the document" or "Research".

Reply:
Fixed